src/uwtools/api/config.py

```diff
@@ -24,7 +24,13 @@ def compare(
     config_b_format: str,
 ) -> bool:
     """
-    ???
+    Compare two config files.
+
+    :param config_a_path: Path to first config file.
+    :param config_a_format: Format of first config file.
+    :param config_b_path: Path to second config file.
+    :param config_b_format: Format of second config file.
+    :return: ``False`` if config files had differences, otherwise ``True``.
```

Comment:
Here and below, I've added initial API-specific docstrings.

Unfortunately, there appears to be [a bug with `docformatter`](https://github.com/PyCQA/docformatter/issues/243) whereby it is not correctly formatting docstrings that contain backticks, which we have here for styling in the API docstrings. The issue was resolved, but I've added info showing, I hope, that it is still a problem. I'll keep an eye on the issue and open a new one if it doesn't get a reaction in a couple/few days. On the balance, I think styling in the API HTML docs is so important that I would rather lose `docformatter` formatting for the API docs than lose styling. For now, we'll have to manually wrap docstring lines in the `uwtools.api.*` modules, but at least `pylint` will remind us to do so if the lines grow too long.